extensions/amp-story/1.0/media-tasks.js

```diff
@@ -445,3 +445,55 @@ export class SwapOutOfDomTask extends MediaTask {
     return true;
   }
 }
+
+/**
+ * Swaps a media element into the DOM, in the place of a placeholder element.
+ */
+export class ManageBitrateTask extends MediaTask {
+  /**
+   * @param {!BitrateManager} bitrateManager The manager.
+   */
+  constructor(bitrateManager) {
+    super('manage-bitrate');
+
+    /** @private @const {!Element} */
+    this.bitrateManager_ = bitrateManager;
+  }
+
+  /** @override */
+  executeInternal(mediaEl) {
+    this.bitrateManager_.manage(mediaEl);
+    return Promise.resolve();
+  }
+
+  /** @override */
+  requiresSynchronousExecution() {
+    return true;
```

Comment:
Do these actually require synchronous execution?  What would happen if these were executed asynchronously, but still "immediately" (i.e. in the next frame)

Reply:
No need to require sync